.github/workflows/continous-integration-os.yml

```diff
@@ -53,7 +53,7 @@ jobs:
       matrix:
         # TODO: Windows was removed for now, see https://github.com/ZcashFoundation/zebra/issues/3801
         os: [ubuntu-latest, macos-latest]
-        rust: [stable]
+        rust: [stable, beta]
```

Comment:
I'd like to only run beta Rust on Linux:
- the concurrent macOS runner limit is much lower than the Linux limit
- macOS is slower than Linux
- macOS is a second-tier Zebra support platform
- there shouldn't be much difference between the macOS and Linux build or test results

Reply:
Fixed by [d79e10f](https://github.com/ZcashFoundation/zebra/pull/4637/commits/d79e10f180a6611a3d07298a621beb51283127e2)